egs2/aishell/asr1/run_streaming.sh

```diff
@@ -20,7 +20,7 @@ use_wordlm=false
 # (train_set will be "${train_set}_sp" if speed_perturb_factors is specified)
 speed_perturb_factors="0.9 1.0 1.1"
 
-./asr.sh                                               \
+./asr_streaming.sh                                               \
```

Comment:
If my idea of including the inference part in  `asr_streaming.sh` would work, then we can change here to

```
./asr.sh
...
...

./asr_streaming.sh
...
...
```